scalafix-tests/input/src/main/scala/test/escapeHatch/AnchorExpression.scala

```diff
@@ -37,7 +37,7 @@ object EscapeHatchExpression {
 
   (
     null,
-    1 + "foo",
+    Some(1) + "foo",
     a.get
   ) // scalafix:ok NoInfer.any2stringadd, Disable.get
 
```

Comment:
this is the test where `NoInfer.any2stringadd` does not get triggered but still passes due to the bug in the regex

Reply:
Aaah, lol. Nice catch!

Reply:
Yes indeed. But the bug in the regex was preventing it from happening